documentation/api/io.strimzi.api.kafka.model.KafkaJmxOptions.adoc

```diff
@@ -1,6 +1,6 @@
 Configures JMX connection options.
 
-JMX metrics are obtained from Kafka brokers, ZooKeeper nodes, Kafka Connect, and MirrorMaker 2.0 by opening a JMX port on 9999.
+Open a JMX port on `9999` to obtain JMX metrics from Kafka brokers, ZooKeeper nodes, Kafka Connect, and MirrorMaker 2.0.
```

Comment:
I replied to the original review comment ... but I guess you missed it. This does not work because it suggests user should open the port. But that is not the case. If the user configures JXM, the operator will make sure the port is open.

Reply:
Alternatively, you can say _Connect to port 9999_ since that is what user would do.

Reply:
We can word it this way:

> Get JMX metrics from Kafka brokers, ZooKeeper nodes, Kafka Connect, and MirrorMaker 2.0. by opening a JMX port on 9999.

Reply:
Delay on your comments here. :-)

I like your suggestion, so we now have:

> Get JMX metrics from Kafka brokers, ZooKeeper nodes, Kafka Connect, and MirrorMaker 2.0. by connecting to port 9999.